docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md

```diff
@@ -9,6 +9,17 @@ related:
     link: docs/pbc/all/tax-management/page.version/vertex/vertex.html
 ---
 
+## Prerequisites
+
+Before you can integrate Vertex, make sure that your project is ACP-enabled. See [App Composition Platform installation](/docs/acp/user/app-composition-platform-installation.html) for details.
+
+The Vertex app requires the following Spryker modules:
+
+* `spryker/tax-app: ^0.1.0`
+* `spryker-shop/cart-page: ^3.38.0`
+* `spryker/product-offer-availability: ^1.4.0` (Marketplace only)
+* `spryker/merchant-profile: ^1.5.0` (Marketplace only)
+
```

Comment:
I think we need also `"spryker/stock-address": "^1.2.0"`

Reply:
added